### PR TITLE
More conventional search input layout

### PR DESCRIPTION
**Pull request by karevn**

### CODE REVIEW

**Comment by karevn** [comment]
image

**Comment by nukeop** [comment]
What's wrong with current one?

**Comment by karevn** [comment]
Nothing wrong, but the trend is to give the loupe a "secondary-importance" look without too much accent. You can see it in [Dribble shots](https://dribbble.com/search?q=search+boxhttps://dribbble.com/search?q=search+box), macOS search boxes and iTunes. The current button-like one just looked as a third navigational button being next to "back-forward" ones.

And, having the icon on the right hand side is a matter of taste - I'm totally OK with having at at either side.

**Comment by nukeop** [comment]
Now that you've explained it, I think you are correct about it looking misleadingly like the navigation buttons, this is a helpful change. Thank you a lot for the contribution.

File: app/components/SearchBox/index.js [hunk 1]
```diff
@@ -19,7 +19,7 @@ const SearchBox = ({ handleSearch, loading }) => {
           onChange={handleSearch}
           autoFocus
         />
-        {loading ? <FontAwesome name='spinner' pulse /> : null}
+        <FontAwesome name='spinner' pulse className={loading ? 'loading' : undefined} />
```
Comment (nukeop) [inline app/components/SearchBox/index.js]: minor nitpick: could you use `classnames` here? E.g.
```javascript
import cx from 'classnames';

        <FontAwesome 
          name='spinner' 
          pulse 
          className={cx({ loading })} 
          />
```

**Comment by nukeop** [comment]
One small change and we'll be ready to merge.

**Comment by karevn** [comment]
Thanks, I'll commit it tomorrow.

**Comment by karevn** [comment]
Done

**Comment by nukeop** [comment]
Awesome, thank you for the contribution once again.

**Pull request merged**